### PR TITLE
update pull request template

### PR DESCRIPTION
**Pull request by zryfish**
Signed-off-by: Jeff <jeffzhang@yunify.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update PR template file, add **release-note** section and guide. **release-note** is required if PR introduces user-facing change. There is a good guide on how to write a release note, https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,56 @@
...
+If no, just write "None" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+
+For more information on release notes see: https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md
+-->
+```release-note
+
+```
+### Additional documentation, usage docs, etc.:
```
Comment (LinuxSuRen) [inline .github/PULL_REQUEST_TEMPLATE.md:54]: I'd prefer to remove this empty line to have a consistent style.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,56 @@
...
+<!--
+If no, just write "None" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+
+For more information on release notes see: https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md
+-->
+```release-note
+
+```
```
Comment (LinuxSuRen) [inline .github/PULL_REQUEST_TEMPLATE.md:52]: I'd prefer to keep an empty line between two sections.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,29 +1,56 @@
...
-
-**Which issue(s) this PR fixes**:
+<!-- Thanks for sending a pull request! Here are some tips for you:
+
+1. If you want **faster** PR reviews, read how: https://github.com/kubesphere/community/blob/master/developer-guide/development/the-pr-author-guide-to-getting-through-code-review.md
+2. In case you want to know how your PR got reviewed, read: https://github.com/kubesphere/community/blob/master/developer-guide/development/code-review-guide.md
+3. Here are some coding convetions followed by KubeSphere community: https://github.com/kubesphere/community/blob/master/developer-guide/development/coding-conventions.md
+
+-->
+
```
Comment (JohnNiang) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: I think there are two redundant blank lines for removal.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,29 +1,56 @@
...
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind deprecation
+/kind failing-test
+/kind flake
+/kind regression
+-->
+
```
Comment (JohnNiang) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: One redundant line here as above.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (zryfish) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: There are all comments, and not shown in the PR description, adding blank lines to make it more readable.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (zryfish) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Same here

**Pull request merged**